access-control-app/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM centos:7
+
+# Install Java and clean up.
+#RUN curl -LO --cookie 'oraclelicense=accept-securebackup-cookie' 'http://download.oracle.com/otn-pub/java/jdk/7u67-b01/jdk-7u67-linux-x64.rpm' \
```

Comment:
Remove this commented out line from all of the Dockerfiles.

Reply:
Done